hyrax.gemspec

```diff
@@ -67,7 +67,7 @@ EOF
   spec.add_dependency 'active_attr', '~> 0.9.0'
   spec.add_dependency 'redlock', '>= 0.1.2'
   spec.add_dependency 'retriable', '>= 2.9', '< 4.0'
-  spec.add_dependency 'active-fedora', '>= 11.1.3'
+  spec.add_dependency 'active-fedora', '~> 11.2'
```

Comment:
OK, why don't we close this PR, @darrenleeweber, and if you want AF 11.2.0 in Hyku, let's pin it there. Cool?